### ISSUE TITLE
[Accessibility] Package compatibility issues in migrator are not narrated

### ISSUE THREAD

**Issue by PatoBeltran** [NuGet/Home#6831]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

VS version (if appropriate): 15.7 preview 4

## Detailed repro steps so we can see the same problem

1. Open JAWS to start narrating screen.
2. Launch VS and create new net framework project. (packages.config)
3. Install jquery v3.3.1 and Newtonsoft.Json v6.0.4
4. Right click on packages.config and select "Migrate packages.config to PackageReference"

...

## Expected
The list of issues inside "Package compatibility issues" for each package should be narrated, and the user should be able to navigate the list of issues with the keyboard.

## Actual
JAWS only narrates the names of each package and lets you open or close the list, but there is no way to go to each issue with the keyboard, you can only move between package names.

![package compatibility issues](https://user-images.githubusercontent.com/2132567/38759872-de21870a-3f2c-11e8-84de-44941a6b32f2.PNG)
